lib/matrix_sdk/protocols/cs.rb

```diff
@@ -767,6 +767,28 @@ def get_room_state_all(room_id, **params)
     request(:get, :client_r0, "/rooms/#{room_id}/state", query: query)
   end
 
+  # Retrieves number of events that happened just before and after the specified event
+  #
+  # @param room_id [MXID,String] The room to get events from.
+  # @param event_id [String] The event to get context around.
+  # @param limit [Integer] (10) The limit of messages to retrieve
```

Comment:
This should be documented as an `@option` as it's part of the `params` hash just like `:filter`.